lib/common/templates/compose/object.kt.template

```diff
@@ -17,9 +17,10 @@
 
 package <%= file.packageName %>;
 
-<%  // Ideally we would selectively import the correct classes & functions based on whether the applied transforms actually need them. %>
-import androidx.compose.ui.graphics.Color
-import androidx.compose.ui.unit.*
+<%= options.import.map(function(item) {
+    return 'import ' + item
+  }).join('\n')
+%>
 
 object <%= file.className %> {
```

Comment:
Unrelated to this pr, but shouldn't `file.className` be `options.className`?

Reply:
I don't think so, because in the config .json the className is directly on the object file and not under options